variables/_colors.pcss

```diff
@@ -48,6 +48,7 @@
 	--color-background-transparent: rgba(var(--color-background), 0.6);
 	--color-background-secondary: #F7F6F6;
 	--color-box-shadow: rgba(#000000, 0.14);
+	--color-box-shadow-week-multiday: rgba(#141827, 0.12);
```

Comment:
can we use `--color-box-shadow` instead? they seem close enough, and with the opacity it'll be even harder to tell.

Reply:
I have no problem, but that's the color they are using for this box shadow.

Reply:
Of course. Push back a bit and have that discussion if there is a case for reusability, that's a common goal for FE and design.